File: app/src/main/java/com/example/proyectomovil/Models/Sitio.java
package com.example.proyectomovil.Models;

import java.util.ArrayList;

public class Sitio {
    public int id_sitio;
    public String nombre;
    public String descripcion;
    public String direccion;
    public String tipo;
    public double latitud;
    public double longitud;
    public String favorito;
    public String calificacion;
    public ArrayList<Imagenes> imagenes = new ArrayList<>();
}

File: ProyectoMovilTour/app/Sitio.php
<?php

namespace App;

use Illuminate\Database\Eloquent\Model;

class Sitio extends Model
{
    protected $table = 'sitio';
    protected $primaryKey = 'id_sitio';
    protected $fillable = [
    	'nombre',
    	'descripcion',
    	'direccion',
    	'latitud',
        'longitud',
        'tipo',
    ];

    public function imagenes()
	{
		return $this->hasMany(ImagenSitio::class, 'id_sitio');
	}
}
